process-automation/latest/model/connectors/aws.md

```diff
@@ -730,6 +567,49 @@ The output parameters of the Transcribe connector are:
 | transcription | String | *Required.* Transcription result. |
 | webVTT | JSON | *Optional* Subtitles result in Web Video Text Tracks format. |
 
+### Transcribe configuration parameters
+
+The configuration parameters for the Transcribe connector are:
+
+| Parameter | Description |
+|-----------|-------------|
+| AWS_ACCESS_KEY_ID | *Required.* The access key to authenticate against AWS. |
+| AWS_SECRET_KEY | *Required.* The secret key to authenticate against AWS. |
+| AWS_REGION | *Required.* The region of AWS to use the Textract service in. |
```

Comment:
I suggest removing the "in" at the end of the sentence to make this consistent with the other parts of the document.